jest.config.mjs

```diff
@@ -5,6 +5,9 @@ const config = {
   transform: {
     "^.+.tsx?$": ["ts-jest", {}],
   },
+  moduleNameMapper: {
+    '\\.svg$': '<rootDir>/__mocks__/svgMock.js',
```

Comment:
Why we need mock for svg?

Reply:
see https://github.com/zendesk/copenhagen_theme/pull/554/files#r1888290869